tslearn/shapelets.py

```diff
@@ -172,7 +180,7 @@ def grabocka_params_to_shapelet_size_dict(n_ts, ts_sz, n_classes, l, r):
     Returns
     -------
     dict
-        Dictionnary giving, for each shapelet length, the number of such
+        Dictionary giving, for each shapelet length, the number of such
```

Comment:
Perhaps we should do a check here that the maximum shapelet length in the dictionary is smaller than the shortest timeseries in the dataset? Or will the distance function work there as well (but probably with a smaller value)?

We could include this in a test somewhere.

Reply:
OK, yes.